Documentation/guides/messages/xa0115.md

```diff
@@ -7,9 +7,13 @@ property will need to be updated to remove it before they will build
 successfully with newer versions of Xamarin.Android.  The newer armeabi-v7a ABI
 should now be used instead.  The `$(AndroidSupportedAbis)` setting can be found
 in the **Advanced** section of the **Android Options** project properties in
-Visual Studio and **Android Build** project properties in Visual Studio for Mac.
+Visual Studio and the **Advanced** tab of the **Android Build** project
+properties in Visual Studio for Mac.  In cases where the property page does not
+show an **armeabi** checkbox, one way to ensure the old setting has been removed
+is to un-check and re-check one of the other ABIs and save the changes.
 
 Example message:
-- `Invalid value 'armeabi' in $(AndroidSupportedAbis). This ABI is no longer supported. Please update your project properties.`
+
+  * `Invalid value 'armeabi' in $(AndroidSupportedAbis). This ABI is no longer supported. Please update your project properties to remove the old value. If the properties page does not show an 'armeabi' checkbox, un-check and re-check one of the other ABIs and save the changes.`
```

Comment:
I think you should also show an example of the XML in the csproj file. Sometimes that will be clear to devs, because they will view the diff of their csproj in whichever source control they use.